### ISSUE TITLE
GUI renders above everything else, making it impossible to see the difficulty of several zones

### ISSUE THREAD

**Issue by Two-Tone** [ensingm2/saliengame_idler#79]
Example: ![example](https://i.imgur.com/Yqn4k76.png)

There are two solutions.

1 is to draw the GUI below the zone info (I'm not even sure if this is doable since the game itself seems to be entirely self contained), 2 is to make the background of the GUI semi transparent (eg: `#191919cc`).  Option 2 is the easiest solution as it's just a one line change.

This is what option 2 looks like.

![eg](https://i.imgur.com/roteLOf.png)

**Comment by ensingm2** [comment]
The game GUI is on a single layer canvas, so I don't think it's possible (or at the very least easy) to put it between the map & zone overlay. I've had issues with this as well. Couldn't figure out a decent way to do it. setting opacity is alright but not ideal,

**Comment by Two-Tone** [comment]
I assume it's not idea since due to performance concerns?

**Comment by JeanPaulLucien** [comment]
It's solved in my pull request.

**Comment by HankTheDrunk** [comment]
> It's solved in my pull request.

Coming from the guy who starts an issue about his cpu being 50%+ and provides little to no details..
And.. This edit in your cloned repo..
> // @author		ensingm2, JeanPaulLucien

Just fucking lol, I can't wait for this event to be over and i'm sure the REAL developers here feel the same.
Big thanks to ensingm2 & Harest for their exceptional hard work on this in a short period of time.

**Comment by ensingm2** [comment]
@HankTheDrunk calm down. If people want to contribute, let them. If it's not up to snuff, explain why in the pull request.

@Two-Tone Nah, it's just not optimal because it's still a bit unclean/not clear, opacity doesn't have a _massive_ performance hit, in my experience.

**Comment by JeanPaulLucien** [comment]
@HankTheDrunk the problem with CPU is other question. About the string. Yes, I used a changed string for myself in Tampermonkey to see difference in versions. As anyone I use ctrl+a, copy-past, so there is nothing fantastic that something from personal using was copied randomly. Later I checked my pull request an fixed that.

![saliensbotidleer](https://user-images.githubusercontent.com/24779886/42142848-79cd78fa-7dba-11e8-8568-62b74c57e5a0.jpg)
